app/services/userService.js

```diff
@@ -0,0 +1,14 @@
+const User = require('../models').User,
+  logger = require('../logger'),
+  errors = require('../errors');
+
+exports.getOne = email => {
```

Comment:
This function is not being used anywhere outside the service, in those cases you shouldn't be exporting it, you can just declare it like `const getOne = email => {`